images/Dockerfile.kairos-alpine

```diff
@@ -184,15 +188,6 @@ ARG MODEL
 ARG REGISTRY_AND_ORG="quay.io/kairos"
 ARG K3S_VERSION
 ARG TARGETARCH
-ARG OS_NAME=kairos-${VARIANT}-${FLAVOR}-${FLAVOR_RELEASE}
```

Comment:
these changes were just because the files were not updated with the latest changes in the Dockerfile.kairos image. I will try to get back to the base images ticket to avoid having this issues